docs/data-sources/dataset.md

```diff
@@ -23,50 +23,50 @@ data "truenas_dataset" "dataset" {
 
 ### Required
 
-- **dataset_id** (String)
+- `dataset_id` (String)
 
 ### Optional
 
-- **id** (String) The ID of this resource.
+- `id` (String) The ID of this resource.
 
 ### Read-Only
 
-- **acl_mode** (String) Determine how chmod behaves when adjusting file ACLs. See the zfs(8) aclmode property.
-- **acl_type** (String)
-- **atime** (String) When set to 'on' access time for files when they are read is updated. Set to 'off' to prevent producing log traffic when reading files
-- **case_sensitivity** (String) `sensitive` assumes filenames are case sensitive. `insensitive` assumes filenames are not case sensitive. `mixed` understands both types of filenames.
-- **comments** (String) Any notes about this dataset.
-- **compression** (String) Current dataset compression level
-- **copies** (Number)
-- **deduplication** (String) Deduplication can improve storage capacity, but is RAM intensive
-- **encrypted** (Boolean)
-- **encryption_algorithm** (String)
-- **encryption_root** (String)
-- **exec** (String) `on` if processes can be executed from within this dataset.
-- **inherit_encryption** (Boolean)
-- **key_format** (String)
-- **key_loaded** (Boolean)
-- **locked** (Boolean)
-- **managed_by** (String)
-- **mount_point** (String)
-- **name** (String)
-- **origin** (String)
-- **parent** (String)
-- **pbkdf2iters** (Number)
-- **pool** (String)
-- **quota_bytes** (Number)
-- **quota_critical** (Number)
-- **quota_warning** (Number)
-- **readonly** (String)
-- **record_size** (String) Matching the fixed size of data, as in a database, may result in better performance
-- **record_size_bytes** (Number) Matching the fixed size of data, as in a database, may result in better performance
-- **ref_quota_bytes** (Number)
-- **ref_quota_critical** (Number)
-- **ref_quota_warning** (Number)
-- **ref_reservation** (Number)
-- **reservation** (Number)
-- **snap_dir** (String) .zfs snapshot directory visibility.
-- **sync** (String) `standard` uses the sync settings that have been requested by the client software, `always` waits for data writes to complete, and `disabled` never waits for writes to complete.
-- **xattr** (String)
+- `acl_mode` (String) Determine how chmod behaves when adjusting file ACLs. See the zfs(8) aclmode property.
```

Comment:
Yup, seems like it, threw me off at first as well. I only updated the dependencies and ran `go generate`.  Maybe check for yourself in case I'm missing something though.